packages/commonwealth/server/api/threads.ts

```diff
@@ -36,12 +37,14 @@ export const trpcRouter = trpc.router({
   deleteThread: trpc.command(
     Thread.DeleteThread,
     trpc.Tag.Thread,
-    async (_, output) => {
-      // Using track output middleware to invalidate gac
+    async () => {
+      // Using track output middleware to invalidate global activity cache
       // TODO: Generalize output middleware to cover (analytics, gac invalidation, canvas, etc)
-      const gac = GlobalActivityCache.getInstance();
-      gac && (await gac.deleteActivityFromCache(output.thread_id));
-      return undefined;
+      void cache().deleteKey(
+        CacheNamespaces.Query_Response,
+        'GetGlobalActivity_{}', // this is the global activity cache key
```

Comment:
Not blocking right now but hardcoding this key seems fragile. Will break if we change key generation...

Reply:
Yep, that's a problem... we need to figure out a better way